docs/user-guide/gpus.md

```diff
@@ -0,0 +1,136 @@
+---
+assignees:
+- vishh
+title: GPU Support
+---
+
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
+This page describes how users can consume GPUs and the current limitations.
+
+## Pre-requisites
+
+1. Kubernetes nodes have to be pre-installed with Nvidia drivers. Kubelet will not detect Nvidia GPUs otherwise.
+2. A special **alpha** feature gate `Accelerators` has to be set to true across the system: `--feature-gates="Accelerators=true"`.
```

Comment:
Can this be done on GKE?  Is there a link to a doc explaining how to set these flags in a running or new cluster?  kubeadm?  kops?  kube-up?